### PR TITLE
Center graph on active note, plus other tweaks

### PR DESCRIPTION
**Pull request by riccardoferretti**
Selecting a note in the editor will now highlight it in the graph, and center on it.

Also added some UX/style improvements:
- show label for highlighted node regardless of zoom
- show labels earlier (in terms of zoom)
- thinner edges for lighter visual impact of graph
- reduced relative size of nodes

### CODE REVIEW

File: packages/foam-vscode/src/features/dataviz.ts [hunk 1]
```diff
@@ -20,6 +21,17 @@ const feature: FoamFeature = {
         foam.notes.unstable_removeEventListener(onNoteAdded);
       });
 
+      vscode.window.onDidChangeActiveTextEditor(e => {
+        if (e.document.uri.scheme === "file") {
+          const note = foam.notes.getNoteByURI(e.document.uri.fsPath);
+          if (isSome(note)) {
```
Comment (ingalless) [inline packages/foam-vscode/src/features/dataviz.ts:27]: Not for this PR, but I'm noticing this isSome is starting to get used heavily. Perhaps we should try make the returns from functions that currently result in an "isSome" a little more explicit (null for nothing, for example). What do you think?

**Review by ingalless — APPROVED** [review]
I made a comment, but not relevant to this PR. A broader discussion

File: packages/foam-vscode/src/features/dataviz.ts (hunk 1 above)
Comment (riccardoferretti) [reply packages/foam-vscode/src/features/dataviz.ts:27]: IIUC that's what we use it for. e.g. `getNoteByURI` will return `null` if no note was found with that URI.
`isSome` is the way we know that something was returned.
Basically anywhere we would tend to use `!== null` or `!== undefined` we can (and should) use `isSome` instead. The same, in the opposite direction, applies to `isNone`.

Am I understanding your point correctly?

Comment (ingalless) [reply packages/foam-vscode/src/features/dataviz.ts:27]: Ah, yes you do understand correctly. I misunderstood isSome's usage. Thanks for the explanation!

**Pull request merged**